frontend/app/model/Nav.scala

```diff
@@ -19,7 +19,7 @@ object Nav {
   }
 
   val primaryNavigation = List(
-    NavItem("become a supporter", "/supporter?INTCMP=gdnwb_copts_memco_internal_supporter_link", "Become a supporter"),
+    NavItem("become a supporter", "https://support.theguardian.com?INTCMP=gdnwb_copts_memco_internal_supporter_link", "Become a supporter"),
```

Comment:
shall we change replace the `INTCMP` with an `acquisitionData` param? https://support.theguardian.com/?acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22componentId%22%3A%22become_a_supporter_link_on_membership_site%22%7D

Reply:
Updated 👍  thanks for generating it